addsite

```diff
@@ -4,12 +4,12 @@ set -eu
 echo "Adding new site: $1"
 
 echo "Creating images directory"
-docker-compose exec --user application "web" mkdir //var/www/mediawiki/images/docker/$1
```

Comment:
removing the double slash wll make these all fail on some certain windows shells with certain settings enabled (which are quite common)

Reply:
@addshore thx, restored the slashes